extensions/dicom-rt/src/getSourceDisplaySet.js

```diff
@@ -1,38 +1,15 @@
-export default function getSourceDisplaySet(studies, rtStructDisplaySet) {
-  const referencedDisplaySet = _getReferencedDisplaySet(
+import { getReferencedDisplaySet } from '../../../platform/core/src/classes/metadata/StudyMetadata.js';
```

Comment:
We shouldnt have imports like this. You have to export it properly in OHIF core then use import { getRefernecedDisplaySet } from '@ohif/core'

Reply:
done